### PR TITLE
circleci 0.1.10458

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

---

[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

### CODE REVIEW

File: Formula/circleci.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Circleci < Formula
   homepage "https://circleci.com/docs/2.0/local-cli/"
   # Updates should be pushed no more frequently than once per week.
```
Comment (Rylan12) [inline Formula/circleci.rb:4]: We should probably hold off. The last update was 3 days ago (#61942)

**Comment by chenrui333** [comment]
Sounds good. Closing this.

File: Formula/circleci.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/circleci.rb:4]: It actually populates another two PRs today, maybe we should consider remove that line? (there was a time that circleci has less release every day)
#62129 #62132

Comment (chenrui333) [reply Formula/circleci.rb:4]: I am going to pull the usage report.

**Pull request closed without merging**